packages/block-library/src/navigation-submenu/index.php

```diff
@@ -222,7 +222,7 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 
 		if ( strpos( $inner_blocks_html, 'current-menu-item' ) ) {
 			$tag_processor = new WP_HTML_Tag_Processor( $html );
-			while ( $tag_processor->next_tag( array( 'class_name' => 'wp-block-navigation-item__content' ) ) ) {
```

Comment:
I wonder how many websites are targeting the "wrong" selector...

Reply:
Not sure. `class` is not a first party API though but as we know we need to acknowledge the impact on existing Themes.

How has this been handled in the past? It's not like we can throw an error if someone is using that class.

Reply:
I think we just consider the risk and go with it. I can't remember when we've done this recently. Maybe a dev note?

Reply:
See https://github.com/WordPress/gutenberg/issues/67134

Reply:
If I read this right that the `.wp-block-navigation-item__content` class will be removed by this change, it looks like it's [used by a considerable number of themes](https://www.wpdirectory.net/search/01JD89RMRY58KJ6QCFJGKAPHGP) and a few plugins too.

Reply:
@mrwweb it looks like that class is only used to iterate there and to determine where to place the `current-menu-ancestor` class. So, this specific change doesn't mean that the `wp-block-navigation-item__content` class will be removed.

> anyone remember an instance of class changing like this?

I don't.